### DISCUSSION TITLE
Change rbac service sidecar log level

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#579] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

13.4.0

### Description

Hi,
where can I change the log level of the sidecar of rbac-service? In the "Sidecar" section I'm redirected to the "Authorization" section, but in this section I cannot see a way to change the log level.

Thanks

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @elmalakomar at the moment the Rönd sidecar configured by console uses the same LOG_LEVEL variable as the protected container.

If you want them to differ you could use kustomize and write a patch to edit the variable value. If you wish to do it in Console I'd tell you to open a feature request :)

**Reply by elmalakomar** [reply]
thank you @fredmaggiowski !
